covid_moonshot_ml/datasets/utils.py

```diff
@@ -81,9 +30,12 @@ def add_seqres(pdb_in, pdb_out=None):
     pdbfile_lines = [re.sub("CL", "Cl", l) for l in pdbfile_lines]
     # # remove ligand hetatoms
     # pdbfile_lines = [ line for line in pdbfile_lines if 'LIG' not in line ]
+    if seqres_str:
+        pdbfile_lines = [line for line in pdbfile_lines if not "SEQRES" in line]
     pdbfile_contents = "".join(pdbfile_lines)
+
     if not "SEQRES" in pdbfile_contents:
-        pdbfile_contents = MPRO_SEQRES + pdbfile_contents
+        pdbfile_contents = seqres_str + pdbfile_contents
```

Comment:
I think we only want to add the `seqres_str` in the event that something is actually passed in. the way it's currently set up, if there's just no SEQRES header in the PDB file, you'll get an error with the default arguments when trying to add a `bool` and a `str`. something like this maybe:
```suggestion
        pdbfile_contents = seqres_str + "".join(pdbfile_lines)
    else:
        pdbfile_contents = "".join(pdbfile_lines)
```

Reply:
agreed, done here: https://github.com/choderalab/covid-moonshot-ml/pull/58/commits/4486d0fd4a101824941f03ff1cde7e636a3b5c15